src/Routes.js

```diff
@@ -1,13 +1,20 @@
 import React, { Component } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Login from './pages/Login/Login';
+import Main from './pages/Main/Main';
+import Products from './pages/Products/Products';
+import Detail from './pages/Products/Components/Detail';
 
 class Routes extends Component {
   render() {
     return (
       <Router>
         <Switch>
+          <Route exact path="/" component={Main} />
           <Route exact path="/Login" component={Login} />
+          <Route exact path="/Products" component={Products} />
+          <Route exact path="/products/detail" component={Detail} />
```

Comment:
```suggestion
          <Route exact path="/products" component={Products} />
```
- 아래에서는 products로 경로명 사용하고 있는데 위에서는 대문자로 시작하고 있습니다. 소문자로 바꾸는 것이 좋아보입니다.
- /detail 경로는 사용되지 않을 것 같은데 맞다면 제거해주세요.

Reply:
detail이 곧 제품 상세페이지가 될 예정입니다. 테스트 후 변경하겠습니다.